engine/class_modules/sc_death_knight.cpp

```diff
@@ -15050,6 +15221,11 @@ void death_knight_t::apply_affecting_auras( buff_t& buff )
   buff.apply_affecting_aura( talent.sanlayn.frenzied_bloodthirst );
 
   // Deathbringer
+  if ( is_ptr() )
+  {
+    buff.apply_affecting_aura( talent.deathbringer.reapers_onslaught );
+    buff.apply_affecting_aura( talent.deathbringer.bind_in_darkness );
```

Comment:
This doesn't really hurt anything, but at present the talent doesn't modify anything in the buff.